openmetadata-spec/src/main/resources/json/schema/metadataIngestion/objectstore/containerMetadataConfig.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$id": "https://open-metadata.org/schema/metadataIngestion/containerMetadataConfig.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "ObjectStoreContainerConfig",
+  "description": "Definition of the properties contained by an object store container template config file",
```

Comment:
Should we use the more general term storage service instead of object store?

Reply:
I believe that the current `storageService` is the one we want to deprecate. Kind of like the `object` approach to represent blob-like services such as S3 or ADLS. True that the structure will also allow us to eventually ingest "directory" structures, but I appreciate the separation of objects vs any other free storage service